standup_notes/__main__.py

```diff
@@ -133,18 +182,13 @@ def delete_notes(date_to_delete):
     if files_to_delete:
         print("Here are the file to be deleted")
         print(*files_to_delete, sep='\n')
-        while True:
-            result = input("Are you sure you want to delete these files y/n: ")
-            if result.lower() == 'y':
-                for file in files_to_delete:
-                    os.remove(os.path.join(STANDUP_NOTES, file))
+        result = verify_input("Are you sure you want to delete these files y/n: ")
```

Comment:
this change isn't actually part of the ticket this PR is for?

Reply:
never mind! I just noticed you extracted verify_input to be used elsewhere :) looks good!